client/state/jetpack-agency-dashboard/selectors.ts

```diff
@@ -58,3 +59,7 @@ export function hasSelectedLicensesOfType(
 		state.agencyDashboard?.selectedLicenses?.licenses.includes( type )
 	);
 }
+
+export function getSiteMonitorStatus( state: AppState ): SiteMonitorStatus {
```

Comment:
Should we rename this selector to `getSiteMonitorStatuses`, as it might be confusing when we use a singular form?

Reply:
Fixed - https://github.com/Automattic/wp-calypso/pull/71752/commits/38c13f39ef96cba5e12dc2e6a0e879a4ed5fb6f0